flutter/beagle/lib/bridge_impl/beagle_service_js.dart

```diff
@@ -106,4 +110,14 @@ class BeagleServiceJS implements BeagleService {
       handler(action: action, view: view, element: element);
     });
   }
+
+  void _registerOperationListener() {
+    _beagleJSEngine.onOperation((operationName, params) {
+      final handler = customOperations[operationName];
+      if (handler == null) {
+        return;
+      }
+      handler(params);
```

Comment:
This is just a suggestion to make the code a little more concise, feel free to maintain your solution or this approach.
```suggestion
      if (handler != null) {
        handler(params);
      }
```